internal/build/options.go

```diff
@@ -22,3 +24,12 @@ func shouldRemoveImage() bool {
 	}
 	return true
 }
+
+func manifestBuildArgsToDockerBuildArgs(args model.DockerBuildArgs) map[string]*string {
```

Comment:
this is... confusing, since the type of `manifestBuildArgs` is `DockerBuildArgs`...  i'd rename this func? or maybe since it's only used in the one place, just do it inside the `Options` func?